### PR TITLE
Farzin/Floating rate

### PR DESCRIPTION
**Pull request by farzin-fs**

### CODE REVIEW

File: lib/basic_api/generated/change_email_receive.dart [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          error: error,
+          msgType: msgType,
+          reqId: reqId,
+        );
+
+  /// Creates an instance from JSON.
+  factory ChangeEmailResponse.fromJson(Map<String, dynamic> json) =>
+      ChangeEmailResponse(
+        changeEmail:
+            json['change_email'] == null ? null : json['change_email'] == 1,
```
Comment (hamed-fs) [inline lib/basic_api/generated/change_email_receive.dart:27]: I think `json['change_email'] == null ? null :` is redundant and you can just check `json['change_email'] == 1`.

File: lib/basic_api/generated/change_email_receive.dart [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        changeEmail: changeEmail ?? this.changeEmail,
+        echoReq: echoReq ?? this.echoReq,
+        error: error ?? this.error,
+        msgType: msgType ?? this.msgType,
+        reqId: reqId ?? this.reqId,
+      );
+
+  /// Override equatable class.
+  @override
+  List<Object?> get props => <Object?>[];
```
Comment (hamed-fs) [inline lib/basic_api/generated/change_email_receive.dart:70]: what do you think about this?

```suggestion
  List<Object?> get props => <Object?>[changeEmail];
```

File: lib/basic_api/generated/change_email_send.dart [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        newEmail: newEmail ?? this.newEmail,
+        newPassword: newPassword ?? this.newPassword,
+        verificationCode: verificationCode ?? this.verificationCode,
+        passthrough: passthrough ?? this.passthrough,
+        reqId: reqId ?? this.reqId,
+      );
+
+  /// Override equatable class.
+  @override
+  List<Object?> get props => <Object?>[];
```
Comment (hamed-fs) [inline lib/basic_api/generated/change_email_send.dart:78]: same here, what do you think about writing fields explicitly?

**Comment by farzin-fs** [comment]
@hamed-fs It's all auto-generated, Should I change the script or manually touch the generated models? 🤔 
I think the script is changed in `new-deriv-api` if I'm not mistaken.

**Review by havir-fs — APPROVED** [review]

**Review by hamed-fs — APPROVED** [review]

**Pull request merged**